docs/lab-extension-testing.md

```diff
@@ -196,39 +211,16 @@ Some highlights:
 from ideal. In other words, these numbers are not reflective of what a good
 production setup can achieve.
 
-| Method | Name | # requests | # failures | Median response time | Average response time | Min response time | Max response time | Average Content Size | Requests/s |
-| ------ | ---- | ---------- | ---------- | -------------------- | --------------------- | ----------------- | ----------------- | -------------------- | ---------- |
-| GET | /data/v3/grand-bend/applicants | 1 | 0 | 190 | 189 | 189 | 189 | 657 | 1 |
-| POST | /data/v3/grand-bend/applicants | 1 | 0 | 16 | 16 | 16 | 16 | 0 | 1 |
-| DELETE | /data/v3/grand-bend/applicants/{id} | 1 | 0 | 16 | 16 | 16 | 16 | 0 | 1 |
-| GET | /data/v3/grand-bend/applicants/{id} | 1 | 0 | 16 | 16 | 16 | 16 | 869 | 1 |
-| PUT | /data/v3/grand-bend/applicants/{id} | 1 | 0 | 13 | 13 | 13 | 13 | 0 | 1 |
-| POST | /oauth/token | 1 | 0 | 31 | 31 | 31 | 31 | 108 | 1 |
-| None | Total | 6 | 0 | 16 | 47 | 13 | 189 | 272 | 6 |
+| Type     | Name                                                                       | # reqs |    # fails  |    Avg |     Min |    Max  | Med  |   req/s | failures/s|
```

Comment:
This `Name` column seems far wider than needed - can it be reformatted?

You might already know this tip: I find that the VS Code extension "Markdown All in One" does a great job of this when you use the VS Code "format document" command.